index.html

```diff
@@ -4,13 +4,122 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <title>Template</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" 
+        integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" 
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://bootswatch.com/3/united/bootstrap.min.css">
+    <link rel="stylesheet" href="index.css">
+    <title>Awesome Books with ES6</title>
 </head>
 <body>
-    <h1>Hi</h1>
+    <header>
+        <nav class="navbar navbar-default">
+            <div class="container">
+              <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                  <span class="sr-only">Toggle navigation</span>
+                  <span class="icon-bar"></span>
+                  <span class="icon-bar"></span>
+                  <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="#">Awesome Books</a>
```

Comment:
- [ ] Great jo so far 👏🏻 However, it's one of the main project requirements you Display the current date below the navigation bar. Kindly address this 👍🏻 For this requirement, you need to use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time). Package should be added as a .js file to the project. You have to download the library into your project  using npm then, Import it as a module: ``import { DateTime } from "luxon";``

![image](https://user-images.githubusercontent.com/79058364/222006357-5b982526-ec50-419c-bbc1-e6bcbd86729c.png)